requirements.txt

```diff
@@ -1,23 +1,23 @@
 -i https://pypi.python.org/simple
 appdirs==1.4.4
-asn1crypto==1.3.0
+asn1crypto==1.4.0
 attrs==19.3.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-autobahn==20.6.2; python_version >= '3.5'
+autobahn==20.7.1; python_version >= '3.5'
 automat==20.2.0
 base58==2.0.1; python_version >= '3.5'
 bitarray==1.2.2
 blake2b-py==0.1.3
 bytestring-splitter==2.2.0
 cached-property==1.5.1
 certifi==2020.6.20
-cffi==1.14.0
+cffi==1.14.1
 chardet==3.0.4
 click==7.1.2
 coincurve==13.0.0
 colorama==0.4.3
 constant-sorrow==0.1.0a9
 constantly==15.1.0
-cryptography==2.9.2
+cryptography==3.0
```

Comment:
Uhh, maybe we should check what's going on in this new version (pinging @tuxxy )

Reply:
Ohhh - good eye!

Reply:
Looks fine to me.